examples/batch_statistical.ipynb

```diff
@@ -0,0 +1,1030 @@
+   "metadata": {},
+   "source": [
+    "We also have a utility function `monitor_batch_statistical_job` (and the analysis counterpart `monitor_batch_statistical_analysis`) that periodically checks the status and updates a progress bar."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:sentinelhub.api.batch.utils:Batch job has a status CREATED, sleeping for 60 seconds\n",
+      "INFO:sentinelhub.api.batch.utils:Batch job has a status CREATED, sleeping for 60 seconds\n",
+      "INFO:sentinelhub.api.batch.utils:Batch job has a status CREATED, sleeping for 60 seconds\n",
+      "INFO:sentinelhub.api.batch.utils:Batch job has a status ANALYSING, sleeping for 60 seconds\n",
+      "INFO:sentinelhub.api.batch.utils:Batch job has a status ANALYSING, sleeping for 60 seconds\n",
+      "INFO:sentinelhub.api.batch.utils:Batch job is running\n",
+      "Completion percentage: 100%|██████████| 100/100 [05:31<00:00,  3.32s/it]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'id': '6cd43a45-631f-4d11-90df-4a832d6430b0',\n",
+       " 'status': 'DONE',\n",
+       " 'completionPercentage': 100,\n",
+       " 'lastUpdated': '2022-08-11T07:42:07.184984Z',\n",
+       " 'costPU': 6.027410152775339,\n",
+       " 'created': '2022-08-11T07:30:45.985911Z'}"
+      ]
+     },
+     "execution_count": 49,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# To get the full output we must configure the logger first\n",
+    "logging.basicConfig(level=logging.INFO)\n",
+    "\n",
+    "monitor_batch_statistical_job(request, config=config, analysis_sleep_time=60)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Check results\n",
+    "\n",
+    "With the batch statistical request completed we do a quick inspection of the results. A folder with the same name as our request is located at `OUTPUT_PATH` and it contains multiple JSON files with results. The JSON files are named after the id of the row, not after our custom `identifier`, so we have files from `1.json` all the way to `100.json` on our bucket. \n",
+    "\n",
+    "We can transfer them to the local storage with the help of `AwsBatchResults`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 100/100 [00:11<00:00,  8.92it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "LOCAL_FOLDER = \"batch_output\"\n",
+    "\n",
+    "# credentials are passed via config\n",
+    "config = SHConfig()\n",
+    "config.aws_access_key_id = AWS_ID\n",
+    "config.aws_secret_access_key = AWS_SECRET\n",
+    "\n",
+    "batch_results_request = AwsBatchResults(request, data_folder=LOCAL_FOLDER, config=config)\n",
```

Comment:
Here I would recommend that we also specify `feature_ids` parameter. We can write that it is not mandatory but recommended in case of large number of results.